shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -405,30 +426,186 @@ private Drawable getSplashScreenDrawableFromActivityTheme() {
     }
   }
 
+  @Override
+  public void onStart() {
+    super.onStart();
+    Log.d(TAG, "onStart()");
+    flutterEngine.getSystemChannels().lifecycle.appIsInactive();
+  }
+
+  public void onPostResume() {
+    Log.d(TAG, "onPostResume()");
+    // TODO(mattcarroll): what does 'updateAccessibilityFeatures' do? why is it called here?
+    flutterView.updateAccessibilityFeatures();
+    // TODO(mattcarroll): why does the platform plugin care about "post resume"?
```

Comment:
IIRC, it's there to ensure status bar theme (light/dark, colour, etc.) is re-set to what the app wants it to be.

Reply:
it sounds like maybe the goal is to forcibly override changes that may have been made by Fragments...but that suggests a whole new issue, which is, why are we overriding Fragments that intentionally set that?

Reply:
Let's work together tomorrow with Jason to figure out for sure why this was chosen.  I would love to get onPostResume out of the forwarded calls, if possible.